### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by shimastripe**
## WHAT
- Change the method of getting the accept header.
- Insert a line.

### CODE REVIEW

File: _example/db/db.go [hunk 1]
```diff
@@ -17,10 +17,13 @@ import (
 func Connect() *gorm.DB {
 	dir := filepath.Dir("db/database.db")
 	db, err := gorm.Open("sqlite3", dir+"/database.db")
+
```
Comment (dtan4) [inline _example/db/db.go]: This empty line is unnecessary. AFAIK most of Go code do not insert empty line between getting error and judging whether the error is nil.

It is probably clear where the error is derived from.

```go
db, err := gorm.Open("sqlite3", dir+"/database.db")
if err != nil {
// ...
}
```

File: _example/version/version.go [hunk 2]
```diff
@@ -9,22 +9,15 @@ import (
 )
 
 func New(c *gin.Context) (string, error) {
-	var header string
-
-	if len(c.Request.Header["Accept"]) > 0 {
```
Comment (dtan4) [inline _example/version/version.go]: Hmm, this is workaround of `out of index` error, you wrote at #90.

Do you really want to replace this?

Comment (shimastripe) [inline _example/version/version.go]: If `len(c.Request.Header["Accept"])` == 0, `c.Request.Header.Get("Accept")` return "".
So `out of index` error does not occur.

File: _example/db/db.go (hunk 1 above)
Comment (shimastripe) [inline _example/db/db.go]: Thx. I fixed. Plz review again.

File: _example/version/version.go (hunk 2 above)
Comment (dtan4) [inline _example/version/version.go]: Do we no longer have to fetch the first element of Accept header?

Comment (shimastripe) [inline _example/version/version.go]: That's right. This function can properly get either of them: array and strings.

File: _example/version/version.go [hunk 3]
```diff
@@ -33,6 +26,7 @@ func New(c *gin.Context) (string, error) {
 	}
 
 	_, err := strconv.Atoi(strings.Join(strings.Split(ver, "."), ""))
+
```
Comment (dtan4) [inline _example/version/version.go]: Same as above.

File: _example/version/version.go (hunk 2 above)
Comment (dtan4) [inline _example/version/version.go]: OK, I understood.

File: _example/version/version.go (hunk 3 above)
Comment (shimastripe) [inline _example/version/version.go]: Thx! I fixed.

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**